lms/djangoapps/static_template_view/tests/test_views.py

```diff
@@ -5,14 +5,25 @@
 from django.core.urlresolvers import reverse
 from django.test import TestCase
 
+from openedx.core.djangoapps.site_configuration.tests.test_util import with_site_configuration_context
+
 
 class MarketingSiteViewTests(TestCase):
     """ Tests for the marketing site views """
 
+    def setUp(self):
+        """
+        Sets up the test.
+        """
+        from django.test.utils import setup_test_environment
+        setup_test_environment()
+        super(MarketingSiteViewTests, self).setUp()
+
     def _test_view(self, view_name, mimetype):
         resp = self.client.get(reverse(view_name))
         self.assertEqual(resp.status_code, 200)
         self.assertEqual(resp['Content-Type'], mimetype)
+        return resp
```

Comment:
Why this change?

Reply:
It was originally because I was using it as a setup method, but I am no longer using it. Line deleted.